unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarHUD/Legacy/AvatarHUDView.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Runtime.CompilerServices;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+using DCL.Interface;
+using Image = UnityEngine.UI.Image;
+
+[assembly: InternalsVisibleTo("AvatarHUDTests")]
```

Comment:
We should move this to `AssemblyInfo.cs` (again, general feedback).

Reply:
All that code will end in the trash after the new HUDs go live, so I don't really see the point of creating a new file to delete it later.
but I will do what you said for the new implementation